README.md

```diff
@@ -191,12 +191,16 @@ By default toasts are not shown.
     - TODO([#39](https://github.com/jackbsteinberg/std-toast/issues/39)): Do we need values `"top-stretch"`, `"center-stretch"`, and `"bottom-stretch"` as well? Should this stretching be done automatically on mobile?
 The default (if the attribute is omitted or set to an invalid value) is ???.
     - TODO([#13](https://github.com/jackbsteinberg/std-toast/issues/13)): should this positioning be an attribute or a style
-- `closebutton`: a boolean attribute, determining whether an explicit close button is shown. 
-By default toasts do not have a close button.
-See ["Contents"](#contents) and ["Appearance customization"](#appearance-customization) sections for how to customize the close button when it's shown.
+- `closebutton`: allows setting the toast's close button content (e.g. `<std-toast closebutton="Dismiss">`),
+or leaving it up to the user agent (e.g. `<std-toast closebutton>`).
+If this attribute is left absent,
```

Comment:
Nit: I think this sounds clearer but up to you

```
If this attribute is not present,
```

Reply:
I was trying to keep the sense from the previous text that the default is no close button, but maybe that's obvious enough already.